src/domains/field/registry.ts

```diff
@@ -15,6 +15,7 @@ export interface FieldInnerConfig {
   property: string;
   description?: string;
   isNullable?: boolean;
+  isGetter?: boolean;
```

Comment:
I'm quite strongly against having it explicitly here. It looks like you might define it. But it's part of business logic that could propably be calculated 'on the fly' when needed.

Config is what user or system want it to be. Getter is implementation detail, so I'd say it's not part of `config`.

Reply:
like I've mentioned-detecting getter always using `Object.getOwnPropertyDescriptor` is quite costly.